cliquet/schema.py

```diff
@@ -61,6 +61,12 @@ def is_readonly(self, field):
         """
         return field in self.get_option("readonly_fields")
 
+    @classproperty
+    def missing(cls):
+        if cls != ResourceSchema:
```

Comment:
I tried and it doesn't check for the missing properties. Is it possible to set {} as the default value for data but test that {} meets the requirements of the defined schema?